docs/cudf/source/user_guide/guide-to-udfs.ipynb

```diff
@@ -7,13 +7,24 @@
     "# Overview of User Defined Functions with cuDF"
```

Comment:
<blockquote>Functions used with <code>cudf.Series.apply</code> and <code>cudf.DataFrame.apply</code> can be expected to handle nulls using the same rules as the rest of cuDF. In most cases this translates to nulls propagating through unary and binary operations and yielding more nulls.</blockquote><p> </p><p>I think we could simplify while also being a bit more explicit here:</p><p> </p><p>The null value <code>NA</code> an propagates through unary and binary operations. Thus, <code>NA + 1</code>,<code>abs(NA)</code>, and <code>NA == NA</code> all return <code>NA</code>. To make this concrete, let's look at the same example from above, this time using nullable data:</p><p> </p><p>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/10607/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>